core/shared/platform/common/posix/posix_memmap.c

```diff
@@ -138,18 +138,26 @@ os_mmap(void *hint, size_t size, int prot, int flags, os_file_handle file)
 
     /* memory hasn't been mapped or was mapped failed previously */
     if (addr == MAP_FAILED) {
-        /* try 5 times */
-        for (i = 0; i < 5; i++) {
+        /* try 5 times on EAGAIN */
```

Comment:
Seems should be `try 5 times on EAGAIN or EINTR`?

Reply:
Fixed